modules/removing-cso-operator.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * security/pod-vulnerability-scan.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="uninstalling-container-security-operator_{context}"]
+= Uninstalling the {rhq-cso}
+
+To uninstall the Container Security Operator, you must uninstall the Operator and delete the `imagemanifestvulns.secscan.quay.redhat.com` custom resource definition (CRD). Without removing the CRD, image vulnerabilities are still reported on the {product-title} *Overview* page.
```

Comment:
> Without removing the CRD, image vulnerabilities are still reported on the {product-title} *Overview* page.

Do we need to include this? Is there anywhere we can stash this as a known issue? Why brag about a bug?

Reply:
It's not really a bug. Uninstalling an Operator never removes a CRD.

Reply:
But you're right. It might not be worth mentioning. If the intention is to remove the CSO, that's just part of the process.